src/services/constants.ts

```diff
@@ -1,4 +1,4 @@
-const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'https://mynotif-api.herokuapp.com'
+const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'http://localhost:3000'
```

Comment:
Let's drop the default value completely I would say

Reply:
yes deleted